index-handlers/src/test/java/no/sikt/nva/nvi/index/OpenSearchClientTest.java

```diff
@@ -313,6 +314,19 @@ void shouldReturnSearchResultsUsingFilterAndSearchTermCombined(Entry<String, Int
         assertThat(searchResponse.hits().hits(), hasSize(entry.getValue()));
     }
 
+    @Test
+    void shouldReturnSingleDocumentWhenFilteringByCategory() throws InterruptedException, IOException {
+        addDocumentsToIndex(documentFromString("document_pending.json"));
+
+        var searchParameters =
+            defaultSearchParameters().withCategory(CATEGORY).withAffiliations(List.of(NTNU_INSTITUTION_ID)).build();
+
+        var searchResponse =
+            openSearchClient.search(searchParameters);
+
+        assertThat(searchResponse.hits().hits(), hasSize(1));
+    }
+
```

Comment:
Add a inverse test as well. Show that some other document will be excluded when searching for category

Reply:
Done.